src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -1105,7 +1206,7 @@ int processResults(Result result, List<JSModule> modules, B options)
     return SourceCodeEscapers.javascriptEscaper().asFunction();
   }
 
-  void outputSingleBinary() throws IOException {
+  void outputSingleBinary(B options) throws IOException {
     Function<String, String> escaper = null;
```

Comment:
I'm a bit confused about this method. Do you want to output to files and to a json stream in the same compilation?